services/frontend-service/pkg/handler/handle_test.go

```diff
@@ -544,6 +547,7 @@ func TestServer_Handle(t *testing.T) {
 				KeyRing:      tt.KeyRing,
 				AzureAuth:    tt.AzureAuthEnabled,
 			}
+			// TODO: Insert upstream env into the env config of the repo state here
```

Comment:
I am confused. Is this something you have to do later or something?

Reply:
It's not really pertinent since this test tests the functionality fot the handler in the frontend and this comment refers to the ReleaseTrain endpoint in the backend. I decided to remove it